pygmt/src/grdlandmask.py

```diff
@@ -0,0 +1,95 @@
+"""
+grdlandmask - Create a "wet-dry" mask grid from shoreline data base
+"""
+
+import xarray as xr
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    R="region",
+    I="increment",
```

Comment:
`blockmean`, `blockmedian`, `grdfilter`, `surface` and `x2sys_init` all use `spacing` for the same usage, so I recommend using `spacing` rather than `increment` here.
```suggestion
    I="spacing",
```